istioctl/pkg/validate/validate.go

```diff
@@ -34,7 +32,11 @@ import (
 	"istio.io/istio/pilot/pkg/config/kube/crd"
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/serviceregistry/kube/controller"
+	"istio.io/istio/pkg/config"
 	"istio.io/pkg/log"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
```

Comment:
package imports are all messed-up

Reply:
not sure if this comment still applies. The latest seems fine.